src/applications/mhv/secure-messaging/components/ComposeForm/ReplyForm.jsx

```diff
@@ -117,7 +117,7 @@ const ReplyForm = props => {
           subject,
         };
         if (draft && replyToMessageId) {
-          messageData[`${'draft_id'}`] = replyToMessageId; // if replying to a thread that has a saved draft, set a draft_id field in a request body
+          messageData[`${'draft_id'}`] = replyMessage.messageId; // if replying to a thread that has a saved draft, set a draft_id field in a request body

```

Comment:
QUESTION: Should this have gotten changed? It looks like `replyToMessageId` can be a [few different values](https://github.com/department-of-veterans-affairs/vets-website/blob/b8c00c8e2da55bb84fe7453daa09b4ffa873305f/src/applications/mhv/secure-messaging/components/ComposeForm/ReplyForm.jsx#L57).